coderd/workspacebuilds_test.go

```diff
@@ -541,6 +542,7 @@ func TestWorkspaceBuildStatus(t *testing.T) {
 	closeDaemon.Close()
 	template := coderdtest.CreateTemplate(t, client, user.OrganizationID, version.ID)
 	workspace := coderdtest.CreateWorkspace(t, client, user.OrganizationID, template.ID)
+	auditor := audit.NewMock()
```

Comment:
This auditor needs to be added to the `&coderdtest.Options` struct above!

Reply:
That worked; thank you!